test/e2e/aggregatedapi_test.go

```diff
@@ -56,6 +56,9 @@ var _ = ginkgo.Describe("Aggregated Kubernetes API Endpoint testing", func() {
 		})
 
 		ginkgo.BeforeEach(func() {
+			// To prevent an error caused by the non-existent namespace when the serviceAccount is created on the member cluster, wait first.
```

Comment:
```suggestion
			// Wait for namespace present before creating resources in it.
```

Reply:
We don't need to explain the error that occurred before.

Reply:
ok!